tests/src/FunctionalJavascript/ParagraphsPasteJavascriptTestBase.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Drupal\Tests\paragraphs_paste\FunctionalJavascript;
+
+use Drupal\FunctionalJavascriptTests\WebDriverTestBase;
+use Drupal\Tests\paragraphs\FunctionalJavascript\LoginAdminTrait;
+
+/**
+ * Base class for Javascript tests for paragraphs_paste module.
+ *
+ * @package Drupal\Tests\paragraphs_paste\FunctionalJavascript
+ */
+abstract class ParagraphsPasteJavascriptTestBase extends WebDriverTestBase {
+
+  use LoginAdminTrait;
+
+  /**
+   * Modules to enable.
+   *
+   * @var array
+   */
+  protected static $modules = [
+    'paragraphs_paste_test',
+    'field_ui',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $defaultTheme = 'stark';
+
+  /**
+   * Simulate paste event.
+   *
+   * @param string $text
+   *   Text to copy.
+   */
+  public function simulatePasteEvent($text) {
```

Comment:
I think we should add the corresponding paragraphs field here, because if you have multiple paste areas